charts/skypilot/values.yaml

```diff
@@ -310,6 +310,29 @@ prometheus:
     persistentVolume:
       enabled: true
       size: 10Gi
+    # HTTP Service Discovery configuration for external Kubernetes clusters
+    extraScrapeConfigs: |
```

Comment:
Somehow the extraScrapeConfigs are now honored in the final deployment, am I missing something?